### DISCUSSION TITLE
FBA retroarch or fba settings.

### DISCUSSION THREAD

**Discussion by Lupusyonder** [eduardofilo/RG350_adam_image#171] [RetroArch topics]
Hi, I have this image installed on my rg350 and rg280v. Is there any way to change the vertical tate mode for shooters back to horizontal. I cant seem to find any settings for FBA apart from the basic key config etc?
Cheers

**Comment by eduardofilo** [comment]
Do you mean the standalone FBA or the RetroArch cores for CPS1 and CPS2?

**Reply by Lupusyonder** [reply]
I believe the standalone FBA.  The cps2 shooters all play horizontally.

**Reply by eduardofilo** [reply]
OK. Follow this steps. I have done the test with `19XX: The war against destiny` CPS2 game:

1. Select the CPS2 game in `Arcade > CPS` system in SimpleMenu.
2. Press select to change emulator to: `fba-multi.opk`. Press `B` to close the menu.
3. Press `B+A` to open FBA in UX mode (it may take time to open if you have many games).
4. Search the game in the list.
5. Press `A` to open the run settings of the game.
6. Change `Rotate vertical game` to `-90`.
7. Change `Hardware scaling` to `Aspect` if you want to preserve aspect ratio. If you prefer full screen, then maintain that setting to `Fullscreen`.
8. Press `A` again to launch the game.
9. Close the game (press Power to open menu and then `Exit`).
10. Press `Select` two times to exit FBA.
11. Now open the game normally from SimpleMenu. You should see the game opened with the settings done in FBA UX.

**Reply by Lupusyonder** [reply]
Yes, that works perfectly. Thanks for spending your time on this. I really appreciate It.
Thank you
